src/main/java/Todo.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * This is a subclass of task which simulates Todotasks.
+ */
+public class Todo extends Task {
+
+    /**
+     * Class constructor inherits Task constructor.
+     *
+     * @param description Description of this Todotask.
+     */
+    public Todo(String description) {
+        super(description);
+    }
+
+    /**
+     * Override String representation of Todotasks.
+     *
+     * @return Type T and its description.
+     */
+    @Override
+    public String toString() {
+        return "[T]" + super.toString();
+    }
+
+    @Override
+    public String getData() {
+        return "T" + super.getData();
+    }
+}
```

Comment:
Might want to add explanation what getData does? There is no data member in Todo